docusaurus/docusaurus.config.js

```diff
@@ -62,7 +62,6 @@ const config = {
                     sidebarPath: require.resolve('./sidebars.js'),
                     editUrl: 'https://github.com/airbytehq/airbyte/blob/master/docs',
                     path: '../docs',
-                    exclude: ['**/connector-development/config-based/**']
```

Comment:
include /connector-development/config-based/ to the build, but not to the sidebar